catherine_feijoo/class4/hw.rb

```diff
@@ -0,0 +1,23 @@
+
+def my_max(array)
+	if array.length == 0
+		return
+
+  elsif(array.length == 1)
+	  return array[0]
+
+  else
+  	max_val = array[0]
```

Comment:
nice assignment! and (just for fun) you could technically assign *any* object out of the array!

`max_val = array.sample` or `max_val = array.pop`

it doesn't matter because every single object will get compared as you loop the whole data collection!